ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_manipulation.py

```diff
@@ -450,3 +450,41 @@ def test_paddle_broadcast_to(
         x=x[0],
         shape=shape,
     )
+    
+    
+# flip
+@st.composite
+def _dtype_x_axis(draw, **kwargs):
+    dtype, x, shape = draw(helpers.dtype_and_values(**kwargs, ret_shape=True))
+    axis = draw(helpers.ints(min_value=0, max_value=len(shape) - 1))
```

Comment:
arent you just drawing single integers here? [`paddle.flip`](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/flip_cn.html#flip:~:text=Axis%20to%20flip.%20When%20axis%20is%20int%20type%2C%20and%20axis%3C0%2C%20the%20actual%20calculation%20dimension%20is%20ndim(x)%20%2B%20axis%3B%20when%20axis%20is%20list%20or%20tuple)  also supports tuples of ints so that should also be tested for